test/precompile/hedera-token-service/utils.js

```diff
@@ -577,6 +745,18 @@ class Utils {
     }
   }
 
+  static async getAccountBalance(address) {
```

Comment:
I know it's for test purposes only, but we can use the mirror node for balance querying, right?

Reply:
@natanasow yes, we can